babel.config.js

```diff
@@ -1,7 +1,11 @@
 // File required for jest, webpack includes different babel config
 module.exports = {
   presets: [
-    '@babel/preset-env'
+    ['@babel/preset-env', {
+      targets: {
+        browsers: ['last 1 version', 'ie >= 11']
```

Comment:
The default is a little more forgiving than this https://github.com/browserslist/browserslist#full-list
```
> 0.5%, last 2 versions, Firefox ESR, not dead
```

Reply:
I think `babel.config.js` is only used for jest.
webpack defines it's own target, maybe we want to relax it some? I've got no clue how what's put here changes whats imported in the entrypoints.

Reply:
That makes sense. I just picked one to comment on since they both looked the same. Webpack might go find the babel.config.js if it doesn't have it's own but I'm not sure. I know I've had trouble in the past... Ya, idk. I assume it dynamically does something in @babel/polyfill...

Reply:
Idk what's best either, one version just seemed a little strict.

Reply:
https://browserl.ist/?q=last+1+version%2C+ie+%3E%3D+11 has 51.8% global coverage
https://browserl.ist/?q=defaults%2C+ie+%3E%3D+11 and https://browserl.ist/?q=defaults have 91.34% global coverage